File: VB/13_Conversion/ToWordSecondApproach.vb
Imports Spire.Pdf.Conversion

Namespace ToWordSecondApproach
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Specify the file path of the input PDF file
			Dim file As String = "..\..\..\..\..\..\Data\ToDocx.pdf"

			' Create a new instance of PdfToWordConverter and load a PDF file
			Dim converter As New PdfToWordConverter(file)

			' Convert the PDF to Word with flow layout
			converter.SaveToDocx("ToWordConvorter.docx")

			' Dispose the converter
			converter.Dispose()

			' Launch the file
			PDFDocumentViewer("ToWordConvorter.docx")
		End Sub
		Private Sub PDFDocumentViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS/13_Conversion/ToWordSecondApproach.cs
using System;
using System.Windows.Forms;
using Spire.Pdf.Conversion;

namespace ToWordSecondApproach
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Load a PDF document from the specified file path.
            String file = @"..\..\..\..\..\..\Data\ToDocx.pdf";

            // Create a PdfToWordConverter object to convert the PDF to Word with flow layout.
            PdfToWordConverter converter = new PdfToWordConverter(file);

            // Save the converted Word document.
            converter.SaveToDocx("ToWordConvorter.docx");

            // Dispose of the converter object to release any resources used.
            converter.Dispose();

            //Launch the file.
            PDFDocumentViewer("ToWordConvorter.docx");
        }
        private void PDFDocumentViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}
